### PR TITLE
Feature/crawler config normalize

### PR DESCRIPTION
**Pull request by liushuigs**

### CODE REVIEW

File: crawler/cv/util/config.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from os.path import join, abspath, dirname
+from ConfigParser import SafeConfigParser
```
Comment (webeautiful) [inline crawler/cv/util/config.py:2]: `ConfigParser`, is it a new python module ?

File: crawler/cv/util/config.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    def __init__(self):
+        config_file = join(dirname(dirname(dirname(abspath(__file__)))),
+                           'configs', 'dev.config.ini')
+        self.parser = SafeConfigParser()
+        self.parser.read(config_file)
+
+if __name__ == '__main__':
+    config = CvConfig()
+    print config.get('db', 'db_host')
+    print config.get('db', 'db_user')
```
Comment (webeautiful) [inline crawler/cv/util/config.py]: Remove prefix `db_` in line 17 and line 18

File: crawler/cv/util/config.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from os.path import join, abspath, dirname
+from ConfigParser import SafeConfigParser
+
+
+class CvConfig(object):
+    def get(self, item, key):
+        return self.parser.get(item, key)
+        pass
```
Comment (webeautiful) [inline crawler/cv/util/config.py]: Delete it Line 8

File: crawler/cv/util/config.py (hunk 2 above)
Comment (liushuigs) [inline crawler/cv/util/config.py]: sure, 16-19 lines should be removed

File: crawler/cv/util/config.py (hunk 3 above)
Comment (liushuigs) [inline crawler/cv/util/config.py]: good catch

File: crawler/cv/util/config.py (hunk 1 above)
Comment (liushuigs) [inline crawler/cv/util/config.py:2]: nope, ConfigParser is a built-in module of python

File: crawler/cv/pipelines/a36kr.py [hunk 4]
```diff
@@ -33,6 +33,7 @@ def insert(self, item):
                                    "%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, " \
                                    "%s)"
             self.cursor.execute(sql, item.values())
+            self.conn.commit()
```
Comment (liushuigs) [inline crawler/cv/pipelines/a36kr.py:36]: missing this line result in the failure

**Pull request merged**